### DISCUSSION TITLE
about pwa

### DISCUSSION THREAD

**Discussion by firavoyage** [share121/inter-knot#1005] [其他]
@share121 
 do you know why pwa is useful?
- open app in one tap on homescreen
- send notifications
- offline use
now only the first one is implemented well qwq...

哈哈哈

**Comment by share121** [comment]
已经支持 pwa 了

**Comment by isyuah** [comment]
感觉目前pwa没什么用, 这个既没有实时更新也就没有消息提示，并且每次都需要请求github数据，所以也不能离线使用

**Comment by share121** [comment]
唯一的用处是可以添加桌面快捷方式

**Reply by isyuah** [reply]
好耶，是作者
